src/main/java/com/eighthlight/SocketWrapperSpy.java

```diff
@@ -0,0 +1,56 @@
+package com.eighthlight;
```

Comment:
Since this is likely only going to be used for testing, I would probably keep this under the `src/test` package and not the `src/main` package